airflow/cli/cli_config.py

```diff
@@ -1496,15 +1527,14 @@ class GroupCommand(NamedTuple):
         name="test",
         help="Test a connection",
         func=lazy_load_command("airflow.cli.commands.remote_commands.connection_command.connections_test"),
-        args=(ARG_CONN_ID, ARG_VERBOSE),
+        args=(ARG_CONN_ID, ARG_CONN_TYPE_POSITIONAL, ARG_VERBOSE),
```

Comment:
Why didn't we need this before? Was it just missed?

Reply:
This is due to API, I adopted the CLI but I will adopt this behaviour in the API because we don't really need the conn_type in this context. Created a bug issue to fix the behaviour #47646. Thanks for pointing!